fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/dto/ComarchProductDTO.java

```diff
@@ -22,4 +22,12 @@
     private Double saleGrossPrice;
     private Double quantity;
     private Long id;
+
+    public String get$id() {
+        return $id;
```

Comment:
Just to remind you, that we should change that $id field name to something without "$". There could be some issue in the future.

Reply:
Just moved this out, as we will not use it anywhere. Thanks for checking :).